buzsaki_lab_to_nwb/valero/behaviorinterface.py

```diff
@@ -151,7 +151,28 @@ def run_conversion(self, nwbfile: NWBFile, metadata: dict, stub_test: bool = Fal
         sleep_intervals = mat_file["SleepState"]["ints"]
         available_states = [str(key) for key in sleep_intervals.keys()]
 
-        table = TimeIntervals(name="SleepStates", description="Sleep state of the subject.")
+        description_of_states = {
+            "WAKEstate": "Waked and in locomotion",
+            "NREMstate": "Non-REM sleep",
+            "REMstate": "Rapid eye movement sleep",
+            "WAKEtheta": "Wake with theta",
+            "WAKEnontheta": "Wake without theta",
+            "WAKEtheta_ThDt": "Wake with theta, estimated with higher theta/delta ratio",
+            "REMtheta_ThDt": "REM sleep with theta, estimated with higher theta/delta ratio",
+            "QWake_ThDt": "Quiet wakefulness esimated with higher theta/delta ratio",
+            "QWake_noRipples_ThDt": "Quite wakefulness without ripples, estimated with higher theta/delta ratio",
+            "NREM_ThDt": "Non-REM sleep, estimated with higher theta/delta ratio",
+            "NREM_noRipples_ThDt": "Non-REM sleep without ripples, estimated with higher theta/delta ratio",
+        }
+        description = (
+            "Sleep state of the subject."
+            "Estimated using `https://github.com/buzsakilab/buzcode/tree/master/detectors/detectStates/SleepScoreMaster`"
+        )
+        import json
```

Comment:
Can you move this import to global level?

Reply:
Yep, done.